apps/wallet/src/ui/app/hooks/useGetNFTMeta.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { getObjectFields, isSuiObject } from '@mysten/sui.js';
+import { useMemo } from 'react';
+
+import { useGetObjectData } from './useGetObjectData';
+
+type NFTMetadata = {
+    name: string | false;
+    description: string | false;
+    url: string;
+} | null;
```

Comment:
IMO this type should not be nullable, and the hook should return `NFTMetadata | null`